devenv/docker/blocks/jaeger/docker-compose.yaml

```diff
@@ -10,6 +10,11 @@
     ports:
       - "3100:3100"
     command: -config.file=/etc/loki/local-config.yaml
+    # For this to work you need to install the logging driver see https://github.com/grafana/loki/tree/master/cmd/docker-driver#plugin-installation
+    logging:
```

Comment:
@cyriltovena does this make sense? It works but maybe there is an easier way to do this.

Reply:
Alternatively you can add promtail or fluentbit to the stack, but this probably the easiest yes and fastest to setup, plus it adds nice labels for compose.